test/int32.test.js

```diff
@@ -174,6 +386,53 @@ describe('Int32', function() {
     });
   });
 
+  describe('custom casters', () => {
+    const defaultCast = mongoose.Schema.Types.Int32.cast();
+
+    afterEach(() => {
+      mongoose.Schema.Types.Int32.cast(defaultCast);
+    });
```

Comment:
```suggestion
  	afterEach(() => {
      mongoose.Schema.Types.Int32.cast(false);
    });
```

Reply:
Your suggested change means we're turning casting off (setting to `_defaultCast`), but we want to go back to the initial behavior which is `defaultCast` variable here (cast/int32.js's `cast` function)